src/BasisTheory.net/Tokens/Entities/Token.cs

```diff
@@ -19,6 +20,10 @@ public class Token : TokenResponse
         [JsonPropertyName("mask")]
         public dynamic Mask { get; set; }
```

Comment:
With the update on vault to return everything in Data instead of Mask for /tokenize, do we need to update this?

Reply:
Removal of `Mask` should probably be done separately, or deprecated now and then followed up with removal.